packages/rich-text-editor/docs/ToggleIconButton.stories.tsx

```diff
@@ -28,11 +30,68 @@ export default {
 export const DefaultStory = (args: ToggleIconButtonProps) => (
   <ToggleIconButton {...args} />
 )
-
-DefaultStory.args = {
+const defaultButton = {
   label: "Bold",
   icon: boldIcon,
-  isActive: false,
 }
 
+DefaultStory.args = { isActive: false, label: "Bold", icon: boldIcon }
+
+const StickerSheetTemplate: Story<{ isReversed: boolean }> = ({
+  isReversed,
+}) => (
+  <>
+    <StoryWrapper isReversed={isReversed}>
+      <StoryWrapper.RowHeader
+        headings={[
+          "Default",
+          "Secondary",
+          "Primary",
+          "Destructive",
+          "Secondary Destructive",
+        ]}
+      />
+      <StoryWrapper.Row rowTitle="base">
+        <ToggleIconButton isActive={false} {...defaultButton} />
+        <ToggleIconButton
+          mood="secondary"
+          isActive={false}
+          {...defaultButton}
+        />
+        <ToggleIconButton mood="primary" isActive={false} {...defaultButton} />
+        <ToggleIconButton
+          mood="destructive"
+          isActive={false}
+          {...defaultButton}
+        />
+        <ToggleIconButton
+          mood="secondary-destructive"
+          isActive={false}
+          {...defaultButton}
+        />
+      </StoryWrapper.Row>
+      <StoryWrapper.Row rowTitle="Selected">
```

Comment:
Not ideal that we don't have this name consistent between implementation and UI Kit. @jmitsios can we convince you to call this `Active` instead of `Selected`? Reasoning here: https://cultureamp.slack.com/archives/C02NUQ27G56/p1646181677465939?thread_ts=1646181455.286509&cid=C02NUQ27G56

Reply:
> Not ideal that we don't have this name consistent between implementation and UI Kit. @jmitsios can we convince you to call this `Active` instead of `Selected`? Reasoning here: https://cultureamp.slack.com/archives/C02NUQ27G56/p1646181677465939?thread_ts=1646181455.286509&cid=C02NUQ27G56

Consider it done.
These have been updated in the UI Kit now to be `Active` instead :)

Reply:
Updated 🦖